.github/workflows/build-storybook.yml

```diff
@@ -30,5 +33,5 @@ jobs:
         # For a `push` event, the branch is `github.ref_name`.
         if: ${{ (github.head_ref || github.ref_name) == 'main' }}
         run: |
-          git remote add storybook git@github.com:MetaMask/metamask-storybook.git
+          git remote add storybook https://${{ secrets.STORYBOOK_TOKEN }}@github.com/MetaMask/metamask-storybook.git
```

Comment:
Hmm, doesn't this sytnax require a username here as well? e.g. `metamaskbot:${{ token }}`